blog/2024-05-01-gsoc-intern.md

```diff
@@ -0,0 +1,12 @@
+---
+author: Antonin Delpeuch
+title: "Welcome to Zyad, our 2024 Google Summer of Code intern"
```

Comment:
Note: the official terminology from GSoC is "*contributor*", not "intern"

Reply:
> Note: the official terminology from GSoC is "contributor", not "intern"

Correct, and they also don't use the term "internship," because that implies an employment relationship, which GSoC is not and they are careful to avoid any such implication.